Dockerfile-Server

```diff
@@ -0,0 +1,49 @@
+FROM centos:7
+
+WORKDIR /dockermount
+
+RUN yum -y update && yum clean all
+RUN yum -y install wget gcc openssl-devel bzip2-devel libffi libffi-devel zlib-devel
+RUN yum -y groupinstall "Development Tools"
+
+##### Install PostgreSQL 10 client (psql)
+RUN yum -y install https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+RUN yum -y install postgresql10
+
+##### Building python 3.7
+WORKDIR /usr/src
+RUN wget --quiet https://www.python.org/ftp/python/3.7.3/Python-3.7.3.tgz
+RUN tar xzf Python-3.7.3.tgz
+WORKDIR /usr/src/Python-3.7.3
+RUN ./configure --enable-optimizations
+RUN make altinstall
+RUN ln -sf /usr/local/bin/python3.7 /usr/bin/python3
+RUN echo "$(python3 --version)"
+
+##### Copy python packaged
+WORKDIR /dockermount
+COPY requirements/ /dockermount/requirements/
+RUN python3 -m pip install -r requirements/requirements.txt
+
+##### Additional server requirements
+RUN yum install -y \
+    git \
+    gcc-c++ \
+    postgresql-devel \
+    pcre \
+    pcre-devel \
+    mlocate \
+    wget \
+    unzip \
+    bzip2-devel \
+    memcached \
+    libmemcached \
+    http://mirror.centos.org/centos/7/os/x86_64/Packages/libmemcached-devel-1.0.16-5.el7.x86_64.rpm
+RUN python3 -m pip install -r requirements/requirements-server.txt
```

Comment:
`pip install` can take multiple `-r` args for multiple requirements files. I think it may be better to just expand line 26 to take an additional file rather than running pip another time and adding another layer to the docker image.

Reply:
the dependency on memcached and libmemcached being installed before `requirements-server.txt` kind of throws this out the window.

Reply:
resolved in latest patch